src/RimDev.ApplicationInsights.Filters/Processors/RedactQueryStringValueTelemetryProcessorOptions.cs

```diff
@@ -0,0 +1,13 @@
+namespace RimDev.ApplicationInsights.Filters.Processors
+{
+    public class RedactQueryStringValueTelemetryProcessorOptions
+    {
+        /// <summary>What should we replace the QueryString argument value with?</summary>
+        public string RedactedValue { get; set; } = "REDACTED";
+
+        /// <summary>List of QueryString argument names to redact.  Note that in .NET Core,
+        /// those argument names are case insensitive. (See <see cref="QueryHelpers.ParseQuery"/>.)
+        /// </summary>
+        public string[] Names { get; set; }
```

Comment:
I'm good to merge, but wondered if you had thoughts on naming this property `Keys` to align with the framework's naming for querystring helpers.